packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts

```diff
@@ -15,13 +15,22 @@ export const getStyles = (props: ICartesianChartStyleProps): ICartesianChartStyl
         width: '100%',
         height: '100%',
         flexDirection: 'column',
-        overflow: 'hidden',
       },
       className,
     ],
     chartWrapper: {
       overflow: 'auto',
     },
+    tooltipTitle: {
+      textAlign: 'center',
+      fontFamily: 'Segoe UI',
+      fontSize: '10px',
+      fontWeight: '600',
+      fontStyle: 'normal',
```

Comment:
There are tokens for all these properties. Dont set hardcoded values

Reply:
added